### ISSUE TITLE
Drive Encoder "V" Graph for Drive Encoder Angular Ramp

### ISSUE THREAD

**Issue by Monjhall** [acmerobotics/road-runner-quickstart#266]
Using drive encoders, we are receiving the following graphs for angular ramp and track width.

![image](https://github.com/acmerobotics/road-runner-quickstart/assets/762305/ab1da1b9-8cdf-4f0a-86e6-a37c8fa99b24)

![image](https://github.com/acmerobotics/road-runner-quickstart/assets/762305/e5aa2296-c47d-4b06-a92e-ba86acc434d2)

We saw similar behavior in #234, but we're still experiencing this issue using drive motor encoders on angular ramp. This seems to result in an invalid KV and KS, which then cause issue in the next stage of tuning. For that next stage we calculated trackWidthTicks manually, taking our measured track width of 15 inches and multiplying it by our previously determined inPerTicks value.

We've verified all motor and encoder wires are going to the correct ports in our expansion hub, that the correct motors are reversed, and that our inPerTicks and lateralInPerTicks values are close to what we'd expect theoretically.

**Comment by teamhazmat13201** [comment]
We just created a new branch and go the same issue. In our case the inPerTicks were half the value expected, and we got the V pattern on the AngularRampLogger. We had another branch where all of it is working correctly! After a lot of careful comparison, turned out that when we reversed the Encoders in DriveLocalizer() we had copied and pasted the REVERSE-ing lines from MecanumDrive(). But the encoder names are different! In MacanumDrive(), the motors are leftBack and rightBack. In DriveLocalizer(), the encoder names are leftRear and rightRear. Once we corrected it, we had correct inPerTicks and the correct ramp pattern in AngularRampLogger.

Hope this helps.
